spec/models/mail_server_log_spec.rb

```diff
@@ -18,61 +18,62 @@
   describe ".load_file" do
     before :each do
       allow(AlaveteliConfiguration).to receive(:incoming_email_domain).and_return("example.com")
+      allow(AlaveteliConfiguration).to receive(:incoming_email_prefix).and_return('foi+')
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.